converters/toZigbee.js

```diff
@@ -566,11 +566,51 @@ const converters = {
     light_colortemp: {
         key: ['color_temp', 'color_temp_percent'],
         convertSet: async (entity, key, value, meta) => {
+            let colorTempMin;
+            let colorTempMax;
+            if (entity.constructor.name === 'Group' && entity.members.length > 0) {
```

Comment:
This can be simplified to something like this:

```js
const minCandidates = entity.members.map(m => m.getClusterAttributeValue('lightingColorCtrl', 'colorTempPhysicalMin')).filter(v => v !== undefined);
if (minCandidates.length === 0) {
   // log warning here
} else {
  colorTempMin = Math.min(...minCandidates);
}


```

Reply:
Looks good, will try.

Also you made the same logical error I made in the beginning. We need `max` of the colorTempPhysicalMin and `min` of colorTempPhysicalMax for groups. To have the most restrictive range, unless I am wrong.

dev1: min=5, max=25
dev2: min=10, max=50

The range that is valid for everyone is 10-25 not 5-50.

Reply:
Yes, that's right indeed